orc8r/cloud/go/obsidian/swagger/registry.go

```diff
@@ -0,0 +1,33 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package swagger
+
+import (
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/lib/go/registry"
+)
+
+// GetSpecServicers returns all the registered Swagger spec servicers
+func GetSpecServicers() ([]remoteSpec, error) {
```

Comment:
Okay so I just re-read through your `remoteSpec` definition. Two small changes:
- make remoteSpec type public (`RemoteSpec`). Since we're going to be using that type outside the package, we want the type to be public
- add a `GetService() string` method to return the name of the service pointed to

The other existing service mesh examples of this registry pattern return an interface type. But that's because (a) we want to be able to easily mock those objects cause they're more complex and (b) we were refactoring from local objects to remote objects. In this case, (a) we don't need to mock the behavior cause it's simple enough to spin up the backing service and (b) the plugin architecture is dead